Examples/Run/Reconstruction/Common/SeedingExample.cpp

```diff
@@ -121,21 +121,43 @@ int runSeedingExample(int argc, char* argv[],
   };
   seedingCfg.outputSeeds = "seeds";
   seedingCfg.outputProtoTracks = "prototracks";
-  seedingCfg.rMax = 100.;
-  seedingCfg.deltaRMax = 60.;
-  seedingCfg.collisionRegionMin = -250;
-  seedingCfg.collisionRegionMax = 250.;
-  seedingCfg.zMin = -2000.;
-  seedingCfg.zMax = 2000.;
-  seedingCfg.maxSeedsPerSpM = 1;
-  seedingCfg.cotThetaMax = 7.40627;  // 2.7 eta
-  seedingCfg.sigmaScattering = 50;
-  seedingCfg.radLengthPerSeed = 0.1;
-  seedingCfg.minPt = 500.;
-  seedingCfg.bFieldInZ = 0.00199724;
-  seedingCfg.beamPosX = 0;
-  seedingCfg.beamPosY = 0;
-  seedingCfg.impactMax = 3.;
+  seedingCfg.gridConfig.rMax = 100._mm;
+  seedingCfg.seedFinderConfig.rMax = seedingCfg.gridConfig.rMax;
+
+  seedingCfg.seedFilterConfig.deltaRMin = 1_mm;
+  seedingCfg.seedFinderConfig.deltaRMin = seedingCfg.seedFilterConfig.deltaRMin;
+
+  seedingCfg.gridConfig.deltaRMax = 60._mm;
+  seedingCfg.seedFinderConfig.deltaRMax = seedingCfg.gridConfig.deltaRMax;
+
+  seedingCfg.seedFinderConfig.collisionRegionMin = -250_mm;
+  seedingCfg.seedFinderConfig.collisionRegionMax = 250._mm;
+
+  seedingCfg.gridConfig.zMin = -2000._mm;
+  seedingCfg.gridConfig.zMax = 2000._mm;
+  seedingCfg.seedFinderConfig.zMin = seedingCfg.gridConfig.zMin;
+  seedingCfg.seedFinderConfig.zMax = seedingCfg.gridConfig.zMax;
+
+  seedingCfg.seedFilterConfig.maxSeedsPerSpM = 1;
+  seedingCfg.seedFinderConfig.maxSeedsPerSpM =
+      seedingCfg.seedFilterConfig.maxSeedsPerSpM;
+
+  seedingCfg.gridConfig.cotThetaMax = 7.40627;  // 2.7 eta
+  seedingCfg.seedFinderConfig.cotThetaMax = seedingCfg.gridConfig.cotThetaMax;
+
+  seedingCfg.seedFinderConfig.sigmaScattering = 50;
```

Comment:
```suggestion
  seedingCfg.seedFinderConfig.sigmaScattering = 5;
```

Reply:
I think I would rather decouple changing this value from this PR.

Reply:
I agree - besides, we will probably find out that one single value for this is not good enough.